service-commands/src/libs.js

```diff
@@ -237,6 +237,31 @@ function LibsWrapper(walletIndex = 0) {
     return users[0]
   }
 
+  /**
+   * Fetch user metadatas from discprov given list of userIds
+   * @param {number[]} userIds
+   */
+  this.getUserAccount = async wallet => {
+    assertLibsDidInit()
+    const userAccount = await this.libsInstance.discoveryProvider.getUserAccount(wallet)
+
+    if (!userAccount) {
+      throw new Error('No user account found.')
+    }
+
+    return userAccount
+  }
+
+  this.setCurrentUser = async userAccount => {
+    assertLibsDidInit()
+    this.libsInstance.userStateManager.setCurrentUser(userAccount)
+    const creatorNodeEndpoints = userAccount.creator_node_endpoint
+    if (creatorNodeEndpoints) {
+      const primary = creatorNodeEndpoints ? creatorNodeEndpoints.split(',')[0] : ''
```

Comment:
there's a function in libs/src/creatorNode/index.js for `getPrimary`. can we use that?

Reply:
yesssssssir